docs/introduction/get-started/p2p-public-note-txs.md

```diff
@@ -0,0 +1,68 @@
+In this section, we show you how to execute transactions and send funds to another account using the Miden client through public notes. 
```

Comment:
Can you link to the section in the docs, where we explain what public notes are? And also in the `private-p2p` section we can link to that explainer.

https://docs.polygon.technology/miden/miden-base/architecture/notes/#note-storage-mode